hub-saml/src/main/java/uk/gov/ida/saml/core/security/AssertionsDecryptersForValidatedResponse.java

```diff
@@ -0,0 +1,52 @@
+package uk.gov.ida.saml.core.security;
+
+import org.opensaml.saml.saml2.core.Assertion;
+import org.slf4j.LoggerFactory;
+import org.slf4j.event.Level;
+import uk.gov.ida.saml.hub.validators.response.common.UnDecryptedValidatedResponseCounter;
+import uk.gov.ida.saml.security.AssertionDecrypter;
+import uk.gov.ida.saml.security.exception.SamlFailedToDecryptException;
+
+import java.util.List;
+
+public class AssertionsDecryptersForValidatedResponse { 
```

Comment:
Not sure we need the ValidatedResponse suffix on this class

Reply:
I have removed the Suffix